web/src/components/SpeakerCard.vue

```diff
@@ -62,6 +66,10 @@ export default {
       const location = this.prefectures.find((elem) => (elem.id === locationId));
       return location?.fields?.prefecture;
     },
+    languages() {
+      const languageIds = this.speaker.get('languages2');
```

Comment:
For the migration, does this have to be `languages` as well? (instead of `languages2`)

Reply:
Yes, I am going to do a search-and-replace for all languages2.